qt_ui/windows/newgame/QNewGameWizard.py

```diff
@@ -586,6 +589,9 @@ def __init__(self, parent=None):
 
         modLayout = QtWidgets.QGridLayout()
         modLayout_row = 1
+        modLayout.addWidget(QtWidgets.QLabel("AH-64D Apache"), modLayout_row, 0)
```

Comment:
While we're adding it as a mod, it doesn't need to be treated as such. Opting out of mods in the wizard is so that Liberation won't generate a mission you can't even load. There's no risk of that for the apache.

Reply:
Ah you're completly right! Ai can still use the (human flyable) Apache even if the user has not purchased it yet. Thanks my brain was already in sleep :D